lib/workflows/claim_substantiation/state.py

```diff
@@ -175,6 +179,13 @@ class ClaimSubstantiatorState(BaseModel):
         default=None,
         description="Dictionary mapping supporting file indices to their summaries",
     )
+    indexed_collections: Dict[str, str] = Field(
+        default_factory=dict,
+        description="Maps file hashes to collection IDs for RAG retrieval",
+    )
```

Comment:
does that need to live in the state? is it a information we want to store and return?